shardingsphere-infra/shardingsphere-infra-context/src/test/java/org/apache/shardingsphere/infra/context/refresher/type/RenameTableStatementSchemaRefresherTest.java

```diff
@@ -0,0 +1,169 @@
+import org.apache.shardingsphere.infra.metadata.schema.event.SchemaAlteredEvent;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.ddl.table.RenameTableDefinitionSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.generic.table.SimpleTableSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.generic.table.TableNameSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.ddl.RenameTableStatement;
+import org.apache.shardingsphere.sql.parser.sql.common.value.identifier.IdentifierValue;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+import javax.sql.DataSource;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public final class RenameTableStatementSchemaRefresherTest {
+
+    private final String oriTableNamePrefix = "TABLE";
+
+    private final String newTableNamePrefix = "NEW_TABLE";
+
+    private final String schemaMetaDataName = "SCHEMA_META_DATA_NAME";
+
+    private final String databaseName = "DATABASE_NAME";
+
+    private final String logicDataSourceName = "LOGIC_DATA_SOURCE_NAME";
+
+    @Mock
+    private ShardingSphereMetaData schemaMetaData;
+
+    @Mock
+    private FederationDatabaseMetaData database;
+
+    @Mock
+    private Map<String, OptimizerPlannerContext> optimizerPlanners;
+
+    private List<String> logicDataSourceNames;
+
+    @Mock
+    private RenameTableStatement sqlStatement;
+
+    @InjectMocks
+    private ConfigurationProperties props;
+
+    @Mock
+    private ShardingSphereResource shardingSphereResource;
+
+    @Mock
+    private ShardingSphereSchema shardingSphereSchema;
+
+    private RenameTableStatementSchemaRefresher renameTableStatementSchemaRefresher;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
+        renameTableStatementSchemaRefresher = new RenameTableStatementSchemaRefresher();
+    }
+
+    @Test
+    public void assertRefreshNonRenameTableStatement() {
+        refreshRenameTableStatement(0);
+    }
+
+    @Test
+    public void assertRefreshOneRenameTableStatement() {
+        refreshRenameTableStatement(1);
+    }
+
+    @Test
+    public void assertRefreshMultipleRenameTableStatement() {
+        refreshRenameTableStatement(2);
+    }
+
+    @SneakyThrows
+    private void refreshRenameTableStatement(final int renameStatementCount) {
+        Collection<RenameTableDefinitionSegment> renameTables = new LinkedList<>();
+        for (Integer i = 0; i < renameStatementCount; i++) {
```

Comment:
How about use `int` here?

Reply:
the following code uses the Integer.toString() method, so using the Integer, not int.  Thanks.

Reply:
@tianhao960 Hi, I think you don't need to use `Integer.toString()`, just use int directly, you try.

Reply:
hi, @yx9o , you're right. have changed to int. thanks.